Crash after 30 seconds: panic: interface conversion: adapter.Stat is nil, not adapter.SimpleCounterStat

When I run vpp-agent it first appears to communicate with VPP fine, but then after 30 seconds there is an error message looking like this:

```
panic: interface conversion: adapter.Stat is nil, not adapter.SimpleCounterStat

goroutine 87 [running]:
vpp-agent/vendor/git.fd.io/govpp.git/core.(*StatsConnection).GetNodeStats.func2(0xc00075c980, 0x10, 0x10, 0x2, 0x0, 0x0, 0x15efa80)
	/home/bahnhof/go/src/vpp-agent/vendor/git.fd.io/govpp.git/core/stats.go:258 +0x14b
vpp-agent/vendor/git.fd.io/govpp.git/core.(*StatsConnection).GetNodeStats(0xc0005b3100, 0xc00016a070, 0x0, 0x0)
	/home/bahnhof/go/src/vpp-agent/vendor/git.fd.io/govpp.git/core/stats.go:278 +0x7b1
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/govppmux.(*Plugin).GetNodeStats(0x24d3280, 0xc00016a070, 0x0, 0x0)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/govppmux/stats_client.go:52 +0x6a
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/vppcalls.(*TelemetryStats).GetRuntimeInfo(0xc00016a000, 0x17761e0, 0xc000054348, 0x7, 0x24fd280, 0x0)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/vppcalls/telemetry_stats.go:92 +0x54
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry.(*Plugin).updatePrometheus(0x24d2a40, 0x17761e0, 0xc000054348)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/prometheus.go:323 +0x2d46
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry.(*Plugin).periodicUpdates(0x24d2a40)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/telemetry.go:196 +0x15a
created by vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry.(*Plugin).startPeriodicUpdates
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/telemetry.go:181 +0x221
```

The whole output from vpp-agent looks like this:

```
                                      __
 _  _____  ___ _______ ____ ____ ___ / /_
| |/ / _ \/ _ /___/ _ '/ _ '/ -_/ _ / __/
|___/ .__/ .__/   \_'_/\_' /\__/_//_\__/  v0.0.0-dev
   /_/  /_/           /___/

INFO[0000] Starting agent version: v0.0.0-dev            BuildDate= CommitHash= loc="agent/agent.go(134)" logger=agent
INFO[0000] ETCD config not found, skip loading this plugin  loc="etcd/plugin_impl_etcd.go(293)" logger=etcd
INFO[0000] Consul config not found, skip loading this plugin  loc="consul/plugin.go(137)" logger=consul
INFO[0000] Redis config not found, skip loading this plugin  loc="redis/plugin_impl_redis.go(123)" logger=redis
INFO[0000] VPP version: 19.08.1-213~g925b94aa3  loc="govppmux/plugin_impl_govppmux.go(241)" logger=govpp
INFO[0000] Namespace plugin initialized                  loc="nsplugin/ns_plugin.go(110)" logger=linux-nsplugin
INFO[0000] kafka config not found  - skip loading this plugin  loc="kafka/plugin_impl_kafka.go(72)" logger=kafka
INFO[0000] GRPC config not found, skip loading this plugin  loc="grpc/plugin_impl_grpc.go(135)" logger=grpc
INFO[0000] grpc server not available                     loc="orchestrator/orchestrator.go(80)" logger=dispatcher
INFO[0000] Watch starting for 29 registered NB prefixes  loc="orchestrator/orchestrator.go(85)" logger=dispatcher
INFO[0000] Listening on http://0.0.0.0:9191              loc="rest/plugin_impl_rest.go(108)" logger=http
INFO[0000] Agent plugin state update.                    lastErr="<nil>" loc="statuscheck/plugin_impl_statuscheck.go(182)" logger=status-check plugin=govpp state=ok
WARN[0000] already registered for watch interface events: VPPApiError: Invalid registration (-31)  loc="vpp1908/watch_vppcalls.go(83)" logger=vpp-ifplugin.if-handler
INFO[0000] Starting health http-probe on port 9191       loc="probe/plugin_impl_probe.go(74)" logger=probe
INFO[0000] Unable to register prometheus-probe handler, Prometheus is nil  loc="probe/plugin_impl_probe.go(86)" logger=probe
INFO[0000] Serving /metrics on port 9191                 loc="prometheus/plugin_impl_prometheus.go(87)" logger=prometheus
WARN[0000] No registrations, skipping resync             loc="resync/plugin_impl_resync.go(98)" logger=resync
INFO[0000] Agent plugin state update.                    lastErr="<nil>" loc="statuscheck/plugin_impl_statuscheck.go(182)" logger=status-check plugin=VPPAgent state=ok
INFO[0000] Agent started with 38 plugins (took 21ms)     loc="agent/agent.go(179)" logger=agent
panic: interface conversion: adapter.Stat is nil, not adapter.SimpleCounterStat

goroutine 87 [running]:
vpp-agent/vendor/git.fd.io/govpp.git/core.(*StatsConnection).GetNodeStats.func2(0xc00075c980, 0x10, 0x10, 0x2, 0x0, 0x0, 0x15efa80)
	/home/bahnhof/go/src/vpp-agent/vendor/git.fd.io/govpp.git/core/stats.go:258 +0x14b
vpp-agent/vendor/git.fd.io/govpp.git/core.(*StatsConnection).GetNodeStats(0xc0005b3100, 0xc00016a070, 0x0, 0x0)
	/home/bahnhof/go/src/vpp-agent/vendor/git.fd.io/govpp.git/core/stats.go:278 +0x7b1
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/govppmux.(*Plugin).GetNodeStats(0x24d3280, 0xc00016a070, 0x0, 0x0)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/govppmux/stats_client.go:52 +0x6a
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/vppcalls.(*TelemetryStats).GetRuntimeInfo(0xc00016a000, 0x17761e0, 0xc000054348, 0x7, 0x24fd280, 0x0)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/vppcalls/telemetry_stats.go:92 +0x54
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry.(*Plugin).updatePrometheus(0x24d2a40, 0x17761e0, 0xc000054348)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/prometheus.go:323 +0x2d46
vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry.(*Plugin).periodicUpdates(0x24d2a40)
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/telemetry.go:196 +0x15a
created by vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry.(*Plugin).startPeriodicUpdates
	/home/bahnhof/go/src/vpp-agent/vendor/github.com/ligato/vpp-agent/plugins/telemetry/telemetry.go:181 +0x221
```

This happens for the current master branch of vpp-agent (commit e96000c723), and VPP 19.08 with a startup.conf containing only `socksvr { socket-name /run/vpp/api.sock }` and nothing else. The operating system is Ubuntu 18.04.2 and the Go version used is go1.13.3.

Does this happen due to a bug, or am I doing something wrong?

(I tried vpp-agent release 2.3.0 as well, then there is no crash but instead an error message like this after 30 seconds: "ERRO[0030] GetRuntimeInfo failed: invalid stat type for /sys/node/names  loc="telemetry/metrics.go(325)" logger=telemetry")

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -3,7 +3,7 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:b4260c6646412fb433e49cbbe547af8da97b354fb56dba14cf5ae314d6b0ebf2"
+  digest = "1:620040c2717f0cde301ba5df32c372843f4d2efe61f4aa801b25dbde86b7bffe"
   name = "git.fd.io/govpp.git"
   packages = [
     "adapter",
@@ -19,7 +19,7 @@
     "version",
   ]
   pruneopts = "UT"
-  revision = "809b69ea4a90455445c34bbad7d8e5fea5cf3462"
+  revision = "cc80dbcaaaca8bf1b6042fead850d456cf589a4e"
 
 [[projects]]
   branch = "master"

